tests/tests_unit/tests_db_connection_unit/test_kafka_unit.py

```diff
@@ -185,6 +185,36 @@ def test_kafka_empty_addresses(spark_mock):
         )
 
 
+@pytest.mark.parametrize(
+    "arg, value",
+    [
+        ("bootstrap.servers", "kafka.bootstrap.servers_value"),
+        ("security.protocol", "ssl"),
+        ("sasl.mechanism*", "PLAIN"),
+        ("kafka.sasl.jaas.config", "GSSAPI"),
+        ("key.key_value", "key_value"),
+        ("value.value", "value"),
+    ],
+)
+def test_kafka_invalid_extras(spark_mock, arg, value):
+    with pytest.raises(
+        ValueError,
+        match=re.escape("are not allowed to use in a Extra"),
+    ):
+        Kafka(spark=spark_mock, cluster="some_cluster", addresses=["192.168.1.1"], extra={arg: value})
+
+
+@pytest.mark.parametrize(
+    "arg, value",
+    [
+        ("kafka.group.id", "group_id"),
+        ("group.id", "group_id"),
+    ],
+)
+def test_kafka_valid_extras(spark_mock, arg, value):
+    Kafka(spark=spark_mock, cluster="some_cluster", addresses=["192.168.1.1"], extra={arg: value})
```

Comment:
Проверь еще, что префикс действительно отрезается. Что-то вроде:
```suggestion
    extra = Extra.parse({arg: value})
    extra_dict = extra.dict()
    assert extra_dict["group.id"] == "group_id"
```

Reply:
Сам класс Kafka тут не нужен, хватит и KafkaExtra

Reply:
И в тесте выше тоже